web/tools/trace/modtest.html

```diff
@@ -1,7 +1,11 @@
-<form action="modtest.php" method="get"> 
-<p>Modtest Ip Adresse vom Modbusdevice : <input type="text" name="ip" /></p> 
-<p>Startadresse: <input type="text" name="start" /></p> 
-<p>Laenge: <input type="text" name="len"/></p>
-<p>Modbus id (normalerweise 1): <input type="text" name="id"/></p>
-<p>Function 3 read input register else read holding register: <input type="text" name="fun"/></p> 
-<p><input type="submit" /></p> </form>
+
```

Comment:
```suggestion
<!DOCTYPE html>
<html lang="de">
<head>
	<title>Modbus-Test</title>
	<meta charset="UTF-8">
</head>
<body>
```
Es sollte zumindest ein komplettes HTML-Grundgerüst vorhanden sein.